src/components/LenderProfile/LenderInviteesList.vue

```diff
@@ -81,10 +94,15 @@ export default {
 			type: Object,
 			required: true,
 		},
+		isLoading: {
+			type: Boolean,
+			required: true,
+			default: true,
```

Comment:
The `default` props aren't needed on these props, since `required` is now there.